### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
* LoginUrl added
* telegram.utils got it's own sidebar thingy
* Fix for #1207
* Remove deprecated `timed_out_behavior` from `ConversationHandler` docstring

### CODE REVIEW

File: docs/source/telegram.loginurl.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+telegram.Loginurl
```
Comment (jsmnbom) [inline docs/source/telegram.loginurl.rst]: Should be `telegram.LoginUrl`

File: docs/source/telegram.rst [hunk 2]
```diff
@@ -140,8 +141,4 @@ Passport
     telegram.encryptedpassportelement
     telegram.encryptedcredentials
 
-telegram.utils
---------------
-
-.. toctree::
-    telegram.utils
+.. include:: telegram.utils.rst
```
Comment (jsmnbom) [inline docs/source/telegram.rst]: Why this change? Does it render much differently?

File: telegram/ext/conversationhandler.py [hunk 3]
```diff
@@ -57,11 +57,6 @@ class ConversationHandler(Handler):
     a regular text message is expected. You could use this for a ``/cancel`` command or to let the
     user know their message was not recognized.
 
-    The fourth, optional collection of handlers, a ``list`` named :attr:`timed_out_behavior` is
```
Comment (jsmnbom) [inline telegram/ext/conversationhandler.py]: Did we remove this feature completely?

File: docs/source/telegram.rst (hunk 2 above)
Comment (jsmnbom) [reply docs/source/telegram.rst]: Nevermind, it renders in the sidebar thingy. Good good :D

File: telegram/ext/conversationhandler.py (hunk 3 above)
Comment (Bibo-Joshi) [reply telegram/ext/conversationhandler.py]: Yes, see #1344
